### ISSUE TITLE
Fails with ember-cli 0.2.1

### ISSUE THREAD

**Issue by jschilli** [sir-dunxalot/ember-cli-concat#13]
`postprocessTree` is now called with `css` in addition to `test` and `all` per this PR https://github.com/ember-cli/ember-cli/pull/3581

This results in ember-cli-concat emitting duplicate trees which results in:

```
Merge error: file assets/vendorapp.js exists in /Users/jschilling/dev/q2/ngam-git-prep/tmp/tree_merger-tmp_dest_dir-eyBhwD6S.tmp and /Users/jschilling/dev/q2/ngam-git-prep/tmp/caching-writer-dest-dir_uIDQ4L.tmp
Pass option { overwrite: true } to mergeTrees in order to have the latter file win.
```

working on a fix

**Comment by jschilli** [comment]
simple fix for now is to ignore type === 'css' - PR forthcoming.

Correct answer (but not currently possible) is to postprocessTree('css') and return just the css tree - should support your refactoring.

Until https://github.com/ember-cli/ember-cli/issues/3628 is resolved postprocessTree is called too early so you only get `app.css` not app + vendor

**Comment by sir-dunxalot** [comment]
Merged #14
